util/cpu/cpu_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cpu
+
+import (
+	"runtime"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCPUValue(t *testing.T) {
+	Observer := NewCPUObserver()
+	Observer.Start()
+	exit := make(chan struct{})
+	var wg sync.WaitGroup
+	for i := 0; i < 10; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			for {
+				select {
+				case <-exit:
+					return
+				default:
+					runtime.Gosched()
+				}
+			}
+		}()
+	}
+	time.Sleep(30 * time.Second)
```

Comment:
Is this too long for test?

Reply:
It is not the main problem. because this is a test only for this part. so this test cache can be used for a long time.